### DISCUSSION TITLE
In the speedtest integration, the text appears on the graph

### DISCUSSION THREAD

**Discussion by a1ex-ak** [home-assistant/frontend#21318] [Dashboard] [answered]
After updating to 2024.7.1, the text appeared on the chart. Is this a bug or a feature?

![изображение](https://github.com/home-assistant/frontend/assets/121604147/f767f405-befd-4af0-ba7c-9e5fb0fdabaa)

this is how it was in 2024.7.0
![изображение](https://github.com/home-assistant/frontend/assets/121604147/a7b37fdb-468a-4e8f-ba46-76294d0553b6)

**Comment by karwosts** [comment, marked as answer]
See https://github.com/home-assistant/frontend/issues/21317
